src/echo-common/src/echo-router.ts

```diff
@@ -24,7 +24,9 @@ export class EchoRouter {
     }
   }
 
-  public removeRoute(next: { plugin: string; path: string }) {}
+  public removeRoute(plugin: string) {
```

Comment:
This method no longer removes a route but all routes from a plugin, we should either rename the method or change the implementation

Reply:
Ok changed it to be `unregisterRoute(route: EchoRoute)`